pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1421,6 +1421,16 @@ self: super: {
     amqp = dontCheck super.amqp_0_19_1;
   };
 
+  # HsYAML-aeson depends on a more modern version of HsYAML than the one
+  # available in stackage's LTS 14.23
+  HsYAML-aeson = unmarkBroken (super.HsYAML-aeson.override {
+    HsYAML = self.HsYAML_0_2_1_0;
+  });
+
+  stylish-haskell = (unmarkBroken super.stylish-haskell).override {
```

Comment:
Unstead of using `unmarkBroken`, it is easier just to edit `pkgs/development/haskell-modules/hackage-packages.nix` and mark the two derivations as not broken.  (Be sure to also remove the line setting the Hydra platforms as well.)

Note that this shouldn't be done on `master`, but the released branches don't get updates from Hackage anymore, so it is fine to directly edit those files.

Reply:
Ok, done then.